crates/ruff_linter/src/rules/pylint/snapshots/ruff_linter__rules__pylint__tests__PLE2510_invalid_characters.py.snap

```diff
@@ -57,10 +57,8 @@ invalid_characters.py:55:21: PLE2510 [*] Invalid unescaped character backspace,
 52 52 | zwsp_after_multicharacter_grapheme_cluster = "ಫ್ರಾನ್ಸಿಸ್ಕೊ ​​"
 53 53 | zwsp_after_multicharacter_grapheme_cluster = f"ಫ್ರಾನ್ಸಿಸ್ಕೊ ​​"
 54 54 | 
-55    |-nested_fstrings = f'{f'{f''}'}'
-   55 |+nested_fstrings = f'\b{f'{f''}'}'
+55    |-nested_fstrings = f'{f'{f''}'
+   55 |+nested_fstrings = f'\b{f'{f''}'
```

Comment:
Trying to understand, why did the outer `}` get stripped here?

Reply:
Heh, I was just editing my comment about this in the PR summary. I don't quite understand this either yet. These are tests of files with invalid/non-printable characters. Locally I thought what was happening was that anstream was just stripping the invalid chars, but looking at it now in the github diff it looks different. So I need to investigate this a bit more tomorrow.

Reply:
Ohh interesting! Yeah I did see the PR summary prior to writing this, but was trying to figure out what's up here.